ls1intum/Artemis
# `Text exercises`: Allow text assessment when adblocker is running on browser

### Checklist
#### General



### Motivation and Context


Fixes #4989 issue.
### Description

In this PR I changed resource name from ```api/analytics/``` to ```/api/event-insights/``` and therefore adblocker does not block text assessment API endpoint.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Text Exercise

1. Log in to Artemis as student
2. Navigate to text exercise and submit your solution
3. Then log in as instructor with adblocker extension
4. Navigate to assessment of the text exercise
5. Try to asses the exercise, it should asses without error

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


Changed files: 
``` src/test/java/de/tum/in/www1/artemis/TextExerciseAnalyticsIntegrationTest.java ```
``` src/test/javascript/spec/service/text-assessment-analytics.service.spec.ts ```
``` src/test/javascript/spec/service/text-assessment.service.spec.ts ```
